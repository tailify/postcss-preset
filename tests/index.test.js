'use strict';

const postcss = require('postcss');
const processor = postcss([require('./../lib/index.js')]);

const source = '@media (width >= 768px) {}\n';

test('should not contain invalid rules', async () => {
  const result = await processor.process(source, { from: 'foo/bar.css' });
  expect(result).toBeTruthy();
  expect(result.css).toMatchSnapshot();
});
