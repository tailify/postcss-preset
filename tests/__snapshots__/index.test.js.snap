// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should not contain invalid rules 1`] = `
"@media (min-width: 768px) {}
"
`;
